lib/custom_a11y_rules.js

```diff
@@ -156,6 +171,26 @@ var customRules = {
             evaluate: function(node, options) {
                 return axe.utils.isHidden(node);
             },
-        }
+        },
+        {
+            'id': 'section-heading-first-child',
+            'metadata': {
+                'impact': 'minor',
```

Comment:
Ok, I hear you. I think you are addressing the point of failing or not-failing, which makes sense. But I guess I'm wondering, how would a developer know if the issue is minor or critical when running tests? Is there any hint to give that indication (other than the fail or pass overall)?